lib/galaxy/dependencies/pipfiles/default/pinned-requirements.txt

```diff
@@ -144,32 +146,32 @@ python-novaclient==11.0.0
 python-openid==2.2.5 ; python_version < '3.0'
 python-swiftclient==3.6.0
 pytz==2019.1
-pyyaml==5.1
+pyyaml==5.1.1
 rdflib-jsonld==0.4.0
 rdflib==4.2.2
 repoze.lru==0.7
 requests-oauthlib==1.2.0
 requests-toolbelt==0.9.1
-requests==2.21.0
+requests==2.22.0
 requestsexceptions==1.4.0
-rfc3986==1.2.0
+rfc3986==1.3.2
 routes==2.4.1
 rsa==4.0
 ruamel.ordereddict==0.4.13 ; platform_python_implementation == 'CPython' and python_version <= '2.7'
-ruamel.yaml==0.15.92
-s3transfer==0.2.0
+ruamel.yaml==0.15.97
+s3transfer==0.2.1
 scandir==1.10.0 ; python_version < '3.5'
 schema-salad==2.7.20181126142424
 shellescape==3.4.1
 simplejson==3.16.0
 six==1.11.0
-social-auth-core[openidconnect]==3.1.0+gx0
+social-auth-core[openidconnect]==3.1.0
```

Comment:
This was changed directly in this file in commit 14628e1fe7defc066080a903adfe7ec849193eda , so `make update-dependencies` is incorrectly reverting it. Can you apply:
```diff
diff --git a/lib/galaxy/dependencies/pipfiles/default/Pipfile b/lib/galaxy/dependencies/pipfiles/default/Pipfile
index 6741c92561..02777c6b6b 100644
--- a/lib/galaxy/dependencies/pipfiles/default/Pipfile
+++ b/lib/galaxy/dependencies/pipfiles/default/Pipfile
@@ -75,7 +75,7 @@ pyparsing = "*"
 "Fabric3" = "*"
 paramiko = "*"
 python-genomespaceclient = "<2.0"
-social_auth_core = {version = "==3.1.0", extras = ['openidconnect']}
+social_auth_core = {version = "==3.1.0+gx0", extras = ['openidconnect']}
 cloudauthz = "<=0.2.0"
 gxformat2 = "*"
 
```
to `lib/galaxy/dependencies/pipfiles/default/Pipfile` ? Or even pin it to the new 3.2.0 , if that works.

Then re-run `make update-dependencies`.

Reply:
@nsoranzo reverted the change of social-auth-core; however, running `make update-dependencies` keeps dropping `+gx0` (and also updates `sqlalchemy` to current latest version that causes some unrelated issues), so I did revert without running `make update-dependencies`. 

Similarly, running `make update-dependencies` reverts `social-auth-core[openidconnect]==3.2.0` to `3.1.0`. So, I left that version change to be addressed in a different PR.

Reply:
Did you update the correct file `lib/galaxy/dependencies/pipfiles/default/Pipfile` before re-running `make update-dependencies` as in my comment above?
It did the right thing for me.
Also, it would be good to pin `sqlalchemy` to the current version in the same file (again, before re-running `make update-dependencies`).

Reply:
I've pushed a commit implementing these to your branch.

Reply:
I guess we pushed at the same time :) Yes, my mistake, I made the changes to a wrong file. Thanks for fixing it.